packages/UsageAnalysis/files/Test Track/Viewers/Calendar.md

```diff
@@ -0,0 +1,17 @@
+### Calendar viewer
+
+1. Open the **demog** dataset.
+2. Click the **Add viewer** icon, find **Calendar** viewer and press on it. 
+* Expected result: **Calendar** viewer opens without errors in console. 
```

Comment:
> Expected result: **Calendar** viewer opens without errors in console. 

Do we really need to write "expected results" for each step? I think it's safe to assume that "no errors should occur" or "no errors in console" should be true for each step?

Reply:
Yes, you are right that "no errors should occur" is the expected behavior for all steps of all test scenarios. Moving forward, where possible, I'll place the "expected results" only at the end.